lib/ui/src/modules/ui/libs/filters.js

```diff
@@ -1,37 +1,97 @@
-import fuzzysearch from 'fuzzysearch';
+import Fuse from '@hypnosphi/fuse.js';
 import sortBy from 'lodash.sortby';
 
+const searchOptions = {
+  shouldSort: false,
+  tokenize: false,
```

Comment:
I think wi should use `tokenize: true` here. This will allow things like this:
<img width="314" alt="screen shot 2017-08-11 at 23 18 59" src="https://user-images.githubusercontent.com/6651625/29230432-af95248e-7eeb-11e7-9976-bf2526ef136f.png">